File: Codeforces/C++/problemset/Bit++.cpp
// Problem Link: https://codeforces.com/problemset/problem/282/A

#include <iostream>
#include <string>
using namespace std;
int main()
{
    long long n,x = 0;
    string s;
    cin >> n;
    for(int i = 0; i < n; i++)
    {
        cin >> s;
        if(s == "++X" || s == "X++") x++;
        else if(s == "--X" || s == "X--") x--;
    }
    cout << x;
    return 0;
}

File: Codeforces/Kotlin/src/main/kotlin/problems/Bit++.kt
package problems.problems

/* Problem Link: https://codeforces.com/problemset/problem/282/A */

fun main() {
    val n = readln().toInt()
    var x = 0

    for (test in 1..n) {
        val statement = readln()

        if(statement.contains("++")) x++
        else x--
    }

    print(x)
}